expression/expression.go

```
@@ -407,6 +408,18 @@ func NewValuesFunc(ctx sessionctx.Context, offset int, retTp *types.FieldType) *
 	}
 }
 
+// NewIsTrueFunc create a new istrue function.
+func NewIsTrueFunc(ctx sessionctx.Context, castNull bool, arg Expression) *ScalarFunction {
+	fc := &isTrueOrFalseFunctionClass{baseFunctionClass{ast.IsTruth, 1, 1}, opcode.IsTruth, castNull}
+	f, err := fc.getFunction(ctx, []Expression{arg})
+	terror.Log(err)
```

Should we throw this error out?

---

updated